mantidproject/mantid
# Adds SHARP IDF

**Description of work.**
Adds the IDF of a new spectrometer at the ILL: SHARP.



**To test:**
Load the empty instrument of SHARP. 
Open the instrument view.
Pick the pixel 128 and 129 in any of the tubes, make sure that for those *y* has the same absolute value, but opposite signs.

**Merge with the generator PR in geometry: https://github.com/mantidproject/mantidgeometry/pull/175**


*There is no associated issue.*





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
